### ISSUE TITLE
Unable to open user relative links. v22

### ISSUE THREAD

**Issue by AlexMonson** [laurencedawson/sync-for-reddit#1460]
v22.06.15-20:43 (22038)
https://www.reddit.com/r/ukraine/comments/vdfqjg
This post is a link to
`/user/jesterboyd/comments/vdfq6u/cognitive_gap/`
This link fails to open in the app and gives a failure toast message. See video. 
Additionally, after this failure the swipe to go back gesture breaks. Closing the post with the back arrow then opening the post again fixes this. 

https://user-images.githubusercontent.com/8193077/174051208-9daa0ed9-916e-46e5-85b0-6e0e84290412.mp4

**Comment by laurencedawson** [comment]
Sure you didn't disable swipe to return? Working fine here.

**Comment by AlexMonson** [comment]
Yes, you can see in the video that after going back into the post I swipe to return successfully. I use that gesture all the time.
It seems to be temporarily broken after that failure toast is shown.

**Comment by laurencedawson** [comment]
Ah so its only broken after the page fails to open?

**Comment by AlexMonson** [comment]
Yes, that's correct.

This issue was more about supporting the relative user links, but this is a minor bug too I suppose.
